File: 2020/problem-0/dude_0.cpp
#include <iostream>
using namespace std;

int main()
{
    string s1, s2, result;

    // First, Accept input from console
    // Nothing should be printed out for an input
    getline (cin, s1);
    getline (cin, s2);

    // Then process
    result = s1 + " " + s2;

    // Lastly, print only the result to console
    cout << result << endl;

    return 0;
}

File: 2020/problem-0/dude_0.java
import java.util.Scanner;
public class dude_0
{

public static void main(String[] args)
{
    String a, b, c;
    Scanner s = new Scanner(System.in);

    // First, Accept input from console
    // Nothing should be printed out for an input
    a = s.nextLine();
    b = s.nextLine();

    // Then process
    dude_0 obj = new  dude_0();
    c = obj.concat(a, b);

    // Lastly, print only the result to console
    System.out.println(c);
}

String concat(String x, String y)
{
    String z;
    z = x + " " + y;
    return z;
}

}
